File: Java/Easy/WrongAnswer/alternating-digit-sum_3.java
class Solution {
    public int alternateDigitSum(int n) {
        
        int res = 0;
        boolean isPositive = true;
        while (n > 0) {
            if (isPositive) {
                res += n % 10;
            } else {
                res -= n % 10;
            }
            n /= 10;
            isPositive = !isPositive;
        }
        return res;
    }
}

File: C++/Easy/WrongAnswer/alternating-digit-sum_3.cpp
class Solution {
public:
    int alternateDigitSum(int n) {
        
        int sum = 0;
        int sign = 1;
        while(n){
            sum += sign * (n % 10);
            sign = -sign;
            n /= 10;
        }
        return sum;
    }
};